### ISSUE TITLE
Nanopore Primer v Our Primers

### ISSUE THREAD

**Issue by ChrisMantegna** [RobertsLab/resources#1665]
Hello! [This is the paper I am using](https://onlinelibrary.wiley.com/doi/10.1002/edn3.374) to mimic the Nanopore process - this is the most clear of the 3 I was cobbling together. It looks like I should order the library prep kit from Nanopore and primers akin to those in table 1.  Check out Table 1 and Table 3 in the photos below. Does that seem right to you?
![image](https://github.com/RobertsLab/resources/assets/81385272/12db9784-0ef2-4dbe-a49a-b099c76ec538)

**Comment by sr320** [comment]
this paper compares Nanopore to Illumina... if you are asking if you should possibly follow the methods regarding Nanopore sequencing - sure.

**Comment by ChrisMantegna** [comment]
Right, it's actually pretty interesting to see how the processes compare, but my question is if I do in fact need to order both primers and the prep kit from Nanopore based on their processes.

**Comment by sr320** [comment]
Not sure... why did they use both primers - short answer those is order both and decide later... 

text in Q 

> The Nanopore native barcoding amplicons protocol was followed as per the manufacturer's instructions with minor changes (Table [S2](https://onlinelibrary.wiley.com/doi/10.1002/edn3.374#support-information-section)). The reaction volume was halved for all PCR amplicons and reagent volumes for “end-prep” and “native barcode ligation” steps, to make the sequencing more cost-effective. However, for the “adapter ligation and clean-up” and “priming and loading the SpotON flow cell” steps the volumes were kept as per the manufacturer's instructions. Two barcodes were used per sample so that each barcode represented a different COI primer set, which otherwise would not be able to be called out separately when demultiplexing.



I would think you need everything in S2 
![image](https://github.com/RobertsLab/resources/assets/4442039/ead8bb1a-d233-40f0-86a2-58b41e8f01c9)

**Comment by ChrisMantegna** [comment]
I agree - I also went back through my mountain of bookmarks and found a [protocol to modify](https://www.protocols.io/view/metabarcoding-minion-library-prep-using-custom-pri-bpa7mihn?step=34). I haven't vetted where it came from, but it's looking like the full version of what's been in papers
